Sources/UIKit/UIKit+Helpers.swift

```diff
@@ -56,18 +56,27 @@ extension UITraitCollection {
                 }
             }
         #endif
-        // `UIApplication.shared` is not a valid object in unit tests. Fall back
-        // to a default value if the delegate is nil.
-        if UIApplication.shared.delegate != nil {
-            return UIApplication.shared.preferredContentSizeCategory
-        }
-        else {
+
+        #if NS_EXTENSION_UNAVAILABLE_IOS
+            // `UIApplication.shared` is not a valid object in unit tests. Fall back
+            // to a default value if the delegate is nil.
+            if UIApplication.shared.delegate != nil {
+                return UIApplication.shared.preferredContentSizeCategory
+            }
+            else {
```

Comment:
I think you could drop the `else` and the `#else`, because if we make it into this `#if`, we will return, and if not, we always want to return `UIContentSizeCategory.large`.